pkg/controllers/management/kontainerdrivermetadata/data.go

```diff
@@ -32,7 +32,7 @@ const (
 
 const (
 	APIVersion           = "management.cattle.io/v3"
-	RancherVersionDev    = "2.4"
+	RancherVersionDev    = "2.4.3-rc100"
```

Comment:
Required as the minimum rancher version for k8s 1.18 is set to 2.4.3-rc99 in KDM:

https://github.com/aiyengar2/kontainer-driver-metadata/blob/afe5b43a12112a86eccee8281f89fd4d2e50ae4e/rke/k8s_version_info.go#L121

Reply:
Then the value should be 2.4.3-rc99 I think

Reply:
From offline discussion 2.4.3-rc100 was still showing 1.18 version, but changing ti to 2.4.3-rc99 seems correct

Reply:
Updated in the latest commit!